tokens/staking/stake.md

```diff
@@ -56,6 +54,23 @@ There are many extrinsics related to the staking pallet, so all of them are not
  - **nominatorBondMore**(*address* collator, *uint256* more) — extrinsic to increase the amount of staked tokens for an already nominated collator
  - **revokeNomination**(*address* collator) — extrinsic to remove an existing nomination
 
+## Retrieving Staking Parameters {: #retrieving-staking-parameters } 
+
+You can now read any of the current parameters around staking, such as the ones previously listed in the [General Definitions](#general-definitions) section and more, directly from PolkadotJS Apps.
```

Comment:
Can you check if we are consistent in the way we write PolkadotJS Apps? Now I like to use Polkadot.js Apps but probably back then I was using PolkadotJS Apps.